src/main.cpp

```diff
@@ -231,20 +239,24 @@ void main(void)
 		PRINT("Baudrate detected: %d\n", baudrate);
 
 	uart_irq_callback_set(dev, interrupt_handler);
-	usbSetupDone = true;
-	
+		
 	//reset head and tails values to 0
 	curie_shared_data->cdc_acm_shared_rx_buffer.head = 0;
 	curie_shared_data->cdc_acm_shared_rx_buffer.tail = 0;
 	curie_shared_data->cdc_acm_shared_tx_buffer.head = 0;
 	curie_shared_data->cdc_acm_shared_tx_buffer.tail = 0;
+	usbSetupDone = true;
 }
 
-extern "C" void baudrateReset(void)
+extern "C" void baudrate_reset(void)
 {
-	PRINT("baudrateReset task\r\n");
+	PRINT("baudrate_reset task\r\n");
```

Comment:
Do these prints always spit something out on the UART? or are they only enabled in a debug build or something?